src/tests/run.py

```diff
@@ -66,7 +66,7 @@
     import urllib.request
 
 # Import coreclr_arguments.py from src\coreclr\scripts
-sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "scripts"))
+sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "coreclr", "scripts"))
```

Comment:
There are a few references to "runtest.py" in this script that should be updated to match the renaming.

Reply:
Updated in 2nd commit, thanks for reminding me!